lib/util.js

```diff
@@ -403,6 +403,21 @@ const Util = {
      * 1Mb in bytes
      */
     MB: 1024 * 1024,
+    /**
+     * @description Return a random integer from a given range
+     * @param {number} min
+     * @param {number} max
+     * @returns {number}
+     */
+    secureGetRandomInt(min, max) {       
```

Comment:
You can just use `crypto.randomInt()`

https://nodejs.org/docs/latest-v20.x/api/crypto.html#cryptorandomintmin-max-callback